### PR TITLE
simplify build

### PR DESCRIPTION
**Pull request by larsrh**
Currently, the 2.11 build doesn't work locally for me. I'm simplifying some things to try to figure out what's wrong.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -7,38 +7,96 @@ organization in ThisBuild := "org.typelevel"
...
 
-addCommandAlias("validateJS", ";catsMtlJS/compile;testsJS/test;js/test")
+addCommandAlias("validateJS", ";testsJS/compile;testsJS/test")
 
 addCommandAlias("validate", ";clean;validateJS;validateJVM")
 
 addCommandAlias("gitSnapshots",
                 ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\"")
 
-sbtPlugin := true
```
Comment (larsrh) [inline build.sbt]: This line was what caused the issue. *\*shakes fist\**

**Comment by larsrh** [comment]
I have a follow-up PR for 2.13.0-M5, and then we can get a release going.

File: build.sbt (hunk 1 above)
Comment (kubukoz) [reply build.sbt]: Damn, this line...

Comment (larsrh) [reply build.sbt]:
```scala
val total_hours_wasted_here = 3
```

**Pull request merged**